www/assets/js/components/SearchPage.tsx

```diff
@@ -16,74 +16,72 @@ import SearchFilterDrawer from "./SearchFilterDrawer"
 import Loading, { Spinner } from "./Loading"
 
 import { search } from "../lib/api"
-import { searchResultToLearningResource } from "../lib/search"
 import {
-  COURSENUM_SORT_FIELD,
+  courseSearchResultToLearningResource,
+  resourceSearchResultToLearningResource
+} from "../lib/search"
+import {
   CONTACT_URL,
   SEARCH_COMPACT_UI,
   DEFAULT_UI_PAGE_SIZE,
   COMPACT_UI_PAGE_SIZE,
-  OCW_PLATFORM
+  OCW_OFFEROR
 } from "../lib/constants"
 import { emptyOrNil, isDoubleQuoted } from "../lib/util"
-import { FacetManifest, LearningResourceResult } from "../LearningResources"
+import { FacetManifest } from "../LearningResources"
+import { CourseResource, ContentFile } from "../open_api_generated/api"
```

Comment:
Here and elsewhere that you use types only, I would do
```suggestion
import type { CourseResource, ContentFile } from "../open_api_generated/api"
```
which makes intent clear, and also 100% guarantees there is no impact on runtime.